app/helpers/providers_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ProvidersHelper
+  def provider_back_link(text = 'Back')
```

Comment:
**[nit pick/not a blocker]** Consider setting the default to something that's configured in a locale? This is tying the code to English only.

Reply:
Sorry - I missed that. I'll slip it into the work I'm doing next.